homeassistant/components/proxmoxve/config_flow.py

```diff
@@ -0,0 +1,161 @@
+"""Config Flow for ProxmoxVE."""
+import logging
+
+import proxmoxer
+from requests.exceptions import ConnectTimeout, SSLError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.data_entry_flow import FlowResult
+
+from . import ProxmoxClient
+from .const import CONF_REALM, DEFAULT_PORT, DEFAULT_REALM, DEFAULT_VERIFY_SSL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ProxmoxVEConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """ProxmoxVE Config Flow class."""
+
+    VERSION = 1
+
+    def __init__(self):
+        """Init for ProxmoxVE config flow."""
+        super().__init__()
+
+        self._config = {}
+        self._proxmox_client = None
```

Comment:
Make this class instance, so you don't need a constructor.

Additionally, I think these should not be class/instance variables but passed along when needed.